desc/compute/_metric.py

```diff
@@ -2088,3 +2088,562 @@ def _cvdrift0(params, transforms, profiles, data, **kwargs):
+    + "derivative w.r.t toroidal cylidrical coordinate",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e^rho", "e^theta_PEST", "e^rho_p|PEST", "e^theta_PEST_p|PEST"],
+)
+def _g_sup_rv_p_PEST(params, transforms, profiles, data, **kwargs):
+    data["g^rv_p|PEST"] = dot(data["e^rho"], data["e^theta_PEST_p|PEST"]) + dot(
+        data["e^rho_p|PEST"], data["e^theta_PEST"]
+    )
+    return data
+
+
+@register_compute_fun(
+    name="g^rv_v|PEST",
+    label="\\partial_{\\vartheta} g_{\\zeta\\zeta}|PEST",
+    units="m^{2}",
+    units_long="square meters",
+    description="Radial/Poloidal element of contrvariant metric tensor"
+    + "derivative w.r.t poloidal PEST coordinate",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e^rho", "e^theta_PEST", "e^rho_v|PEST", "e^theta_PEST_v|PEST"],
+)
+def _g_sup_rv_v_PEST(params, transforms, profiles, data, **kwargs):
+    data["g^rv_v|PEST"] = dot(data["e^rho"], data["e^theta_PEST_v|PEST"]) + dot(
+        data["e^rho_v|PEST"], data["e^theta_PEST"]
+    )
+    return data
+
+
+# TODO: Generalize for a general phi before #568
+@register_compute_fun(
+    name="g^rz_p|PEST",
+    label="\\partial_{\\phi} g_{\\zeta\\zeta}|PEST",
+    units="m^{2}",
+    units_long="square meters",
+    description="Radial/Toroidal element of contravariant metric tensor"
+    + "derivative w.r.t toroidal cylindrical coordinate",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e^rho", "e^zeta", "e^rho_p|PEST", "e^zeta_p|PEST"],
+)
+def _g_sup_rz_p_PEST(params, transforms, profiles, data, **kwargs):
+    data["g^rz_p|PEST"] = dot(data["e^rho"], data["e^zeta_p|PEST"]) + dot(
+        data["e^rho_p|PEST"], data["e^zeta"]
+    )
+    return data
+
+
+@register_compute_fun(
+    name="g^rz_v|PEST",
+    label="\\partial_{\\vartheta} g_{\\zeta\\zeta}|PEST",
+    units="m^{2}",
+    units_long="square meters",
+    description="Radial/Toroidal element of contrvariant metric tensor"
+    + "derivative w.r.t toroidal PEST coordinate",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e^rho", "e^rho_v|PEST", "e^zeta_v|PEST", "e^zeta"],
+)
+def _g_sup_rz_v_PEST(params, transforms, profiles, data, **kwargs):
+    data["g^rz_v|PEST"] = dot(data["e^rho"], data["e^zeta_v|PEST"]) + dot(
+        data["e^rho_v|PEST"], data["e^zeta"]
+    )
+    return data
+
+
+################################################################################
+#############--------------JACOBIAN DERIVATIVES PEST---------------#############
+################################################################################
+
+
+@register_compute_fun(
+    name="sqrt(g)_PEST_r|PEST",
+    label="\\partial_{\\rho} \\sqrt{g}_PEST",
+    units="m^{3}",
+    units_long="cubic meters",
+    description="Jacobian determinant of flux coordinate system",
```

Comment:
PEST coordinate system, not general flux coordinates. Same for others below.

Also need to add a note about this being the radial derivative